decidim-conferences/app/commands/decidim/conferences/admin/update_conference.rb

```diff
@@ -0,0 +1,133 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Conferences
+    module Admin
+      # A command with all the business logic when creating a new participatory
+      # conference in the system.
+      class UpdateConference < Rectify::Command
+        # Public: Initializes the command.
+        #
+        # conference - the Conference to update
+        # form - A form object with the params.
+        def initialize(conference, form)
+          @conference = conference
+          @form = form
+        end
+
+        # Executes the command. Broadcasts these events:
+        #
+        # - :ok when everything is valid.
+        # - :invalid if the form wasn't valid and we couldn't proceed.
+        #
+        # Returns nothing.
+        def call
+          return broadcast(:invalid) if form.invalid?
+          update_conference
+
+          if @conference.valid?
+            broadcast(:ok, @conference)
+          else
+            form.errors.add(:hero_image, @conference.errors[:hero_image]) if @conference.errors.include? :hero_image
+            form.errors.add(:banner_image, @conference.errors[:banner_image]) if @conference.errors.include? :banner_image
+            broadcast(:invalid)
+          end
+        end
+
+        private
+
+        attr_reader :form, :conference
+
+        def update_conference
+          @conference.assign_attributes(attributes)
+          save_conference if @conference.valid?
+        end
+
+        def save_conference
+          transaction do
+            update_conference_registrations
+            @conference.save!
+            send_notification_registrations_enabled if should_notify_followers_registrations_enabled?
+            send_notification_update_conference if should_notify_followers_update_conference?
+            schedule_upcoming_conference_notification if start_date_changed?
+            Decidim.traceability.perform_action!(:update, @conference, form.current_user) do
+              @conference
+            end
+          end
+        end
+
+        def update_conference_registrations
+          @conference.registrations_enabled = form.registrations_enabled
+
+          if form.registrations_enabled
+            @conference.available_slots = form.available_slots
+            @conference.registration_terms = form.registration_terms
+          end
+        end
+
+        def attributes
+          {
```

Comment:
`form.attributes`?

Reply:
I do not think so.

Reply:
Sorry! I forgot part of the comment.

We are creating a hash with the attributes and using 'assign_attributes' method. https://apidock.com/rails/v3.2.3/ActiveRecord/AttributeAssignment/assign_attributes